src/boards/include/boards/olimex_rp2350_xl.h

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+// -----------------------------------------------------
+// NOTE: THIS HEADER IS ALSO INCLUDED BY ASSEMBLER SO
+//       SHOULD ONLY CONSIST OF PREPROCESSOR DIRECTIVES
+// -----------------------------------------------------
+
+// This header may be included by other board headers as "boards/olimex_rp2350_xl.h"
+
+#ifndef _BOARDS_OLIMEX_PICO2_XL
+#define _BOARDS_OLIMEX_PICO2_XL
+
+pico_board_cmake_set(PICO_PLATFORM, rp2350)
+
+// Check https://github.com/OLIMEX/RP2350-PICO2-XXL
+
+// For board detection
+#define OLIMEX_PICO2_XL
+
+// --- RP2350 VARIANT ---
+#define PICO_RP2350A 0
+
+// --- UART ---
+#ifndef PICO_DEFAULT_UART
+#define PICO_DEFAULT_UART 0
+#endif
+#ifndef PICO_DEFAULT_UART_TX_PIN
+#define PICO_DEFAULT_UART_TX_PIN 0
+#endif
+#ifndef PICO_DEFAULT_UART_RX_PIN
+#define PICO_DEFAULT_UART_RX_PIN 1
+#endif
+
+// --- LED ---
+#ifndef PICO_DEFAULT_LED_PIN
+#define PICO_DEFAULT_LED_PIN 25
+#endif
+
+// --- I2C ---
+#ifndef PICO_DEFAULT_I2C
+#define PICO_DEFAULT_I2C 0
```

Comment:
Did you mean to use I2C bus 1 here?

Reply:
According to https://github.com/OLIMEX/RP2350-PICO2-XXL/blob/main/DOCUMENTS/RP2350-PICO2-XXL-user-manual.pdf (page7) is I2C 0.

Reply:
It's 1 :)